sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ConditionalPreconditionsTest.java

```diff
@@ -651,6 +657,846 @@ void testDescriptionModificationModificationNoDescriptors() {
         reset(mockManipulations);
     }
 
+    /**
+     * Tests whether DescriptionModificationMdsDescriptorPrecondition correctly calls manipulations
+     * in case a RemovableDescriptor is present.
+     */
+    @Test
+    @DisplayName("DescriptionModificationMdsDescriptorPrecondition correctly calls manipulation when a "
+            + "RemovableDescriptor is present.")
+    public void testDescriptionModificationMdsDescriptorPreconditionGoodPresentDescriptors() {
+
+        // given
+        final var descriptor1Handle = "superHandle1";
+
+        final var presenceMap = new HashMap<>(Map.of(descriptor1Handle, false));
```

Comment:
Shouldn't the descriptor be initially present?

Reply:
Indeed, Corrected.